cms/static/sass/partials/_variables.scss

```diff
@@ -41,11 +41,16 @@ $transparent: rgba(0,0,0,0); // used when color value is needed for UI width/tra
 
 // +Colors - UXPL new pattern library colors
 // ====================
-$uxpl-blue-base: rgba(0, 116, 180, 1); // wcag2a compliant
-$uxpl-blue-hover-active: lighten($uxpl-blue-base, 7%); // wcag2a compliant
+$uxpl-blue-base: rgb(0, 117, 180); // wcag2a compliant
```

Comment:
@alisan617's PR isn't landing for awhile, but the file she created could be checked in earlier. Take a look at #13052 and see what you think. If it doesn't make sense to do as part of this PR, I can create a technical debt story to remove the duplication.